client/src/components/App.jsx

```diff
@@ -1,14 +1,158 @@
 import React, { Component } from 'react';
+import Main from './Main';
+import SideBar from './SideBar';
 
 class App extends Component {
   constructor(props) {
-    super(props)
+    super(props);
 
+    this.state = {
+      details: {
```

Comment:
At first I tried setting it as an empty object, but the data wasn't showing when being pass down to the Main component. That is because the GET request is a async function, and the page renders before receiving the data. So if I set up the object and the properties to empty string, data was able to pass in successfully!